Comp2023/src/main/java/frc/robot/commands/WristMoveToAngle.java

```diff
@@ -0,0 +1,56 @@
+
+// ROBOTBUILDER TYPE: Command.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ARMConsts.WristAngle;
+import frc.robot.subsystems.Arm;
+
+/**
+ *
+ */
+public class WristMoveToAngle extends CommandBase
+{
+  private final Arm  m_wrist;
+  private WristAngle m_Angle;
+
+  public WristMoveToAngle(Arm wrist, WristAngle Angle)
+  {
+    m_wrist = wrist;
+    m_Angle = Angle;
+
+    setName("ClimberMoveToAngle");
+    addRequirements(m_wrist);

```

Comment:
Requiring the Arm subsystem in the elbow and wrist may be a problem in running those two at the same time. I don't have a solution right now, but it will need to be addressed at some point.

Reply:
zara

Reply:
Marking as resolved on Alex's PR--will be handled on Zara's.